core/trino-spi/src/main/java/io/trino/spi/connector/ColumnHandle.java

```diff
@@ -13,6 +13,16 @@
  */
 package io.trino.spi.connector;
 
+/**
+ * ColumnHandle should follow:
```

Comment:
First sentence of javadoc should tell what a thing is. Move this to the second paragraph/sentence